doc/nrf/links.txt

```diff
@@ -391,19 +391,23 @@
 
 .. _`Nordic Semiconductor website`: https://www.nordicsemi.com/
 
-.. _`Nordic nRF9161 DK`: https://www.nordicsemi.com/Products/nRF9161
+.. _`Nordic nRF9161 DK`:
+.. _`nRF9161 DK product page`: https://www.nordicsemi.com/Products/nRF9161
 .. _`nRF9161 DK Downloads`: https://www.nordicsemi.com/Products/nRF9161/Download?lang=en#infotabs
 
 .. _`Nordic nRF9160 DK`:
 .. _`nRF9160 DK product page`: https://www.nordicsemi.com/Products/Development-hardware/nrf9160-dk/
 .. _`nRF9160 DK Downloads`: https://www.nordicsemi.com/Products/Development-hardware/nRF9160-DK/Download#infotabs
 .. _`SUPL client download`: https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF9160-DK/Download#supl-c
 
+.. _`nRF9161 product website`: https://www.nordicsemi.com/Products/nRF9161
```

Comment:
This looks like the same target as Lines 394 - 395, does it make sense to have it as a separate entry? For the nRD9160, there's the difference between the SiP and the DK for the links.

Reply:
The link is available now and is added to the PR.